src/DssSpellCollateralOnboarding.sol

```diff
@@ -35,92 +31,69 @@ contract DssSpellCollateralOnboardingAction {
     // A table of rates can be found at
     //    https://ipfs.io/ipfs/QmTRiQ3GqjCiRhh1ojzKzgScmSsiwQPLyjhgYSxZASQekj
     //
-    uint256 constant THREE_POINT_FIVE_PCT = 1000000001090862085746321732;
+    uint256 constant NUMBER_PCT = 1000000001234567890123456789;
 
     // --- Math ---
     uint256 constant THOUSAND   = 10 ** 3;
     uint256 constant MILLION    = 10 ** 6;
 
-    // --- PRE-REQUISITE GENERAL DEPLOYS ---
-    address constant CDP_REGISTRY              = 0xBe0274664Ca7A68d6b5dF826FB3CcB7c620bADF3;
-    address constant PROXY_ACTIONS_CROPPER     = 0xa2f69F8B9B341CFE9BfBb3aaB5fe116C89C95bAF;
-    address constant PROXY_ACTIONS_END_CROPPER = 0xAa61752a5Abf86A527A09546F23FE8bCB8fAB2C4;
-    address constant CROPPER                   = 0x8377CD01a5834a6EaD3b7efb482f678f2092b77e;
-    address constant CROPPER_IMP               = 0xaFB21A0e9669cdbA539a4c91Bf6B94c5F013c0DE;
-
     // --- DEPLOYED COLLATERAL ADDRESSES ---
-    address constant ETHSTETH                  = 0x06325440D014e39736583c165C2963BA99fAf14E;
-    address constant PIP_ETHSTETH              = 0x0A7DA4e31582a2fB4FD4067943e88f127F70ab39;
-    address constant MCD_JOIN_ETHSTETH_A       = 0x82D8bfDB61404C796385f251654F6d7e92092b5D;
-    address constant MCD_CLIP_ETHSTETH_A       = 0x1926862F899410BfC19FeFb8A3C69C7Aed22463a;
-    address constant MCD_CLIP_CALC_ETHSTETH_A  = 0x8a4780acABadcae1a297b2eAe5DeEbd7d50DEeB8;
-
-    // --- Other addresses ---
-    address constant STETH_PRICE               = 0x911D7A8F87282C4111f621e2D100Aa751Bab1260;
-    address constant WSTETH_MEDIAN             = 0x2F73b6567B866302e132273f67661fB89b5a66F2;
+    address constant XXX                  = 0x06325440D014e39736583c165C2963BA99fAf14E;
+    address constant PIP_XXX              = 0x0A7DA4e31582a2fB4FD4067943e88f127F70ab39;
+    address constant MCD_JOIN_XXX_A       = 0x82D8bfDB61404C796385f251654F6d7e92092b5D;
+    address constant MCD_CLIP_XXX_A       = 0x1926862F899410BfC19FeFb8A3C69C7Aed22463a;
+    address constant MCD_CLIP_CALC_XXX_A  = 0x8a4780acABadcae1a297b2eAe5DeEbd7d50DEeB8;
```

Comment:
can you remove these real addresses too? (I would just hate to have a copy paste from a previous template cause a problem, so making these obviously not addresses feels cleaner to me)

Reply:
done